xamarin/Xamarin.Forms
# Notify child added to CollectionView

### Description of Change ###

Notify child added/removed to CollectionView:
- ChildAdded
- ChildRemoved

### Issues Resolved ### 

- fixes #10454 

### API Changes ###
 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the Issue 10454. Verify that the Label above CollectionView have text (updated using `ChildAdded` event).

### PR Checklist ###
